src/prog_models/utils/calc_error.py

```diff
@@ -380,30 +329,25 @@ def DTW(m, times, inputs, outputs, **kwargs):
 
     Args:
         m (PrognosticsModel): Model to use for comparison
-        times (list[float]): array of times for each sample
-        inputs (list[dict]): array of input dictionaries where input[x] corresponds to time[x]
-        outputs (list[dict]): array of output dictionaries where output[x] corresponds to time[x]
+        times (list[float], list[list[float]]): Array of times for each sample.
+        inputs (list[dict, SimResult]): Array of input dictionaries where input[x] corresponds to time[x].
+        outputs (list[dict, SimResult]): Array of output dictionaries where output[x] corresponds to time[x].
 
     Keyword Args:
-        x0 (StateContainer): Current State of the model
+        x0 (StateContainer, optional): Initial state.
         dt (float, optional): Maximum time step in simulation. Time step used in simulation is lower of dt and time between samples. Defaults to use time between samples.
         stability_tol (float, optional): Configurable parameter.
             Configurable cutoff value, between 0 and 1, that determines the fraction of the data points for which the model must be stable.
             In some cases, a prognostics model will become unstable under certain conditions, after which point the model can no longer represent behavior. 
             stability_tol represents the fraction of the provided argument `times` that are required to be met in simulation, 
-            before the model goes unstable in order to produce a valid estimate of error. 
+            before the model goes unstable in order to produce a valid estimate of mean squared error.
 
             If the model goes unstable before stability_tol is met, NaN is returned. 
-            Else, model goes unstable after stability_tol is met, the error calculated from data up to the instability is returned.
+            Else, model goes unstable after stability_tol is met, the mean squared error calculated from data up to the instability is returned.
```

Comment:
```suggestion
            Else, model goes unstable after stability_tol is met, the error calculated from data up to the instability is returned.
```
Same here, and propagated throughout